src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateClonesArray = [];
```

Comment:
letter 's' in the end of word Clones defines plurality, no need to add array after it
it's either stateArray, which is not really good, or stateClones

but I actually prefer stateHistory